Source/Scene/Cesium3DTile.js

```diff
@@ -206,14 +231,17 @@ define([
         },
 
         /**
-         * Get the tight oriented bounding box
+         * Get the bounding volume of the tile
          *
-         * @type {Promise}
+         * @type {Object}
          * @readonly
          */
-        orientedBoundingBox : {
+        boundingVolume : {
```

Comment:
So it seems [this](https://github.com/e-andersson/cesium/blob/3d-tiles_bounding_volumes/Source/Scene/Points3DTileContentProvider.js#L58-L59) might be a reason that some of the tests for `Points3DTileContentProvider` don't pass, right?